### PR TITLE
[FTLIS042022] matildeCosta pedroCosta

### PR DESCRIPTION
**Pull request by matildedacosta**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,6 +1,113 @@
...
+}
+
+/* Bonus 1:
+Go to lorem ipsum generator and:
+
+Generate 3 paragraphs. Store the text in a variable type of string.
+Make your program count the number of words in the string.
+Make your program count the number of times the Latin word et appears. */
+
+let lorem = "Lorem Ipsum is simply dummy text of the printing and typesetting industry.";
```
Comment (webLisbonTA) [inline js/index.js:73]: Take a look on both your bonus solutions, they are not completely correct.
A hint for Bonus 1 -  You could try the same approach as count Smiles function from the prework

**Review by webLisbonTA — COMMENTED** [review]
Hey! Great job trying all iterations, including bonus!

File: js/index.js [hunk 2]
```diff
@@ -1,6 +1,113 @@
...
+3.2 Print all the characters of the navigator's name, in reverse order. i.e. "nhoJ"
+
+3.3 Depending on the lexicographic order of the strings, print:
+- The driver's name goes first.
+- Yo, the navigator goes first definitely.
+- What?! You both have the same name? */
+
+for (i = 0; i < hacker1.length; i++) {
+    console.log(hacker1[i].toUpperCase())
+}
```
[the remark this answers is not in the document]
Comment (matildedacosta) [reply js/index.js:49]: Ohhh completely missed that part! 
I tried writing the code above but this is what is happening: 

Screenshot 2022-04-12 at 21 13 48


it has something to do with the way I'm looping?

[the remark this answers is not in the document]
Comment (karinaglf) [reply js/index.js:49]: Probably you are console.logging inside the loop

**Pull request closed without merging**